docs/app.md

```diff
@@ -168,6 +169,12 @@ myApp.isRunning();
 myApp.start();
 ```
 
+### App `restart`
```

Comment:
we need to document that this accepts `state` as an option as well as a code example